### ISSUE TITLE
why() misses exception involving complex numbers

### ISSUE THREAD

**Issue by kseistrup** [friendly-traceback/friendly-traceback#109]
```py
In [1]: a = 3+1j
In [2]: b = 1+3j
In [3]: divmod(a, b)
Traceback (most recent call last):
  Code block [3], line 1, in <module>
    divmod(a, b)
  TypeError: unsupported operand type(s) for divmod(): 'complex' and 'complex'
In [4]: why()
No information is known about this exception. Please report […]
```

* Python 3.10.1
* IPython 7.31.0
* friendly_traceback 0.5.10
* friendly 0.5.5

**Comment by aroberge** [comment]
Thanks for the report.

I know that I have **many** TypeError cases not covered yet, but I will try to take care of this one next.

**Comment by kseistrup** [comment]
Take your time, mate. It's not like I run into this exception every day, but I thoght I might as well report it when friendly asked me to.

**Comment by aroberge** [comment]
Fixed in version 0.5.11 for friendly-traceback.

There will be more cases to consider eventually, but this one (with at least one complex number) has been taken care of.
